.github/workflows/test_with_woodwork_main_branch.yml

```diff
@@ -0,0 +1,61 @@
+name: Test Featuretools with Woodwork main Branch
+on:
+  schedule:
+    - cron: "0 * * * *"
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  scheduled_unit_latest_tests:
+    name: ${{ matrix.python_version }} unit tests ${{ matrix.libraries }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python_version: ["3.7", "3.8", "3.9",]
```

Comment:
"3.10" should be added, but we'll want to exclude the "3.10 koalas"  -- see this discussion in the PR that added 3.10 tests

https://github.com/alteryx/featuretools/pull/1940#discussion_r823131888

Reply:
Added 3.10. Using if condition to exclude the "3.10 koalas". Syntax of exclude is working weird when I tested with act locally.